04/components/SpeedTest.js

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useState, useRef } from 'react';
+import Time from './Time';
+import WordsLeft from './WordsLeft';
+import useRandomItem from '../hook';
+
+const wordsList = ['abc', '123', 'fff'];
```

Comment:
Pamiętaj, że takie rozwiązanie może powodować, że jeśli będziesz miał kilka komponentów to każdy z nich będzie operował na tych danych. Jeśli coś dodajesz lub odejmujesz do takiej zmiennej w trakcie działania komponentu lepiej to oprzeć o `state`.  Wtedy w takim ekosystemie jak komponent mamy zamknięte całe rozwiązanie ;)